src/encoded/viewconfigs/news.py

```diff
@@ -0,0 +1,48 @@
+from snovault.viewconfigs.searchview import SearchView
+from encoded.helpers.helper import format_results
+
+
+class NewsView(SearchView):
+    def __init__(self, context, request):
+        super(NewsView, self).__init__(context, request)
+        self.search_term = '*'
+        self.facets = []
+        self.from_ = 0
+        self.size = 25
+        self.es_index = 'page'
+
+    def set_facets(self):
+        if len(self.doc_types) == 1 and 'facets' in self.types[self.doc_types[0]].schema:
+            self.facets.extend(self.types[self.doc_types[0]].schema['facets'].items())
+
+    def preprocess_view(self):
+        self.result['@id'] = '/news/' + self.search_base
+        self.result['@type'] = ['News']
+        self.result['notification'] = ''
+
+        # construct doc_types
+        self.construct_doc_types(['Page'])
+
+        # construct query
+        static_items = [('type', 'Page'), ('news', 'true'), ('status', 'released')]
+        query = self.construct_query(static_items, False)
```

Comment:
construct_query doesn't exist in SearchView or BaseView.   Same with construct_doc_types.  It looks like these functions were removed from SearchView 27 days ago in snovault commit 'phil modifications to kathik refactor' on your 'SNO-31-search-refactoring-v3' branch.   Please go back and make sure inherited method in all the subviews exist in their parents somewhere.

Two things.  

1. Make a list of urls used to test the search refactor feature.  There should be a few for each of the 6 views, search, audit, summary, news, matrix, and report.  Why a few for each?  Each view has conditions, so a good list will hit each condition.  Each time we make an update to the PR we need to check this list first before code review.  It looks like news hasn't worked for a month.

2. Given that all the views eventually inherit from BaseView and all views have similar, required methods.  Good practice would be adding all the functions that must be overridden by a child class to the base class as
```
def construct_query(self, what_ever_args):
    '''
    Contruct Query for such and such
    - Must be overridden by child classes.
    '''
    raise NotImplementedError
```
for example.  This way, when someone creates a new view they know what is required.  Do some googling for NotImplementedError to see how people use it in practice.  It helps to define how parent classes should be implemented.  Add a couple NotImplementedError functions to the base view and then sync up with me to check it out.

Reply:
I did some updates on news.py. Now, you see JSON but not the css-styling. I will address that in an upcoming commit. 

On the two things
1. Okay, I will write it up and place it in Jira; after the addressing the code-feedback

2. I do not fully get this. There is no method that must be overridden.